projects/pipelines/testing/config.yaml

```diff
@@ -33,6 +33,18 @@ ci_presets:
     clusters.sutest.compute.dedicated: false
     clusters.driver.compute.dedicated: false
 
+  stress:
+    extends: [metal]
+    clusters.sutest.compute.dedicated: false
+    clusters.driver.compute.dedicated: false
```

Comment:
the `metal` profile can be automatically applied if [you call this function](https://github.com/openshift-psap/topsail/blob/007f9991e53e6482714859037fa1819d31054202/projects/kserve/testing/test.py#L47C5-L47C77) in the `init`:
```
is_metal = config.ci_artifacts.detect_apply_metal_profile(METAL_PROFILE)
```
and `...dedicated: false` should go to the metal profile

same comment for the mixed preset below

Reply:
Is there a reason to use this:
```
is_metal = config.ci_artifacts.detect_apply_metal_profile(METAL_PROFILE)
```
instead of just extending the `metal` profile?

Reply:
yes! if you extend the `metal` profile, your test won't work well in the cloud. Cloud is great for rehearsal, but also if tomorrow we want to compare the results on AWS